real_time/arduino_src/radio_buggy_mega/main.cpp

```diff
@@ -360,7 +417,6 @@ void brake_init()
 void brake_raise() 
 {
     BRAKE_OUT_PORT |= _BV(BRAKE_OUT_PINN);
-    BRAKE_INDICATOR_PORT &= ~_BV(BRAKE_INDICATOR_PINN);
```

Comment:
Are you asking to have the status LEDs indicate whether the brakes are down or not?